pipeline/common/common_fns.py

```diff
@@ -4,19 +4,42 @@
 
 # COMMAND ----------
 
-dbutils.widgets.dropdown("input_database", "ocio_dex_dev", ["ocio_dex_dev"])
+dbutils.widgets.dropdown("database", "ocio_dex_dev", ["ocio_dex_dev"])
+dbutils.widgets.dropdown("database_checkpoint_prefix", "abfss://ocio-dex-db-dev@ocioededatalakedbr.dfs.core.windows.net/delta/events/", ["abfss://ocio-dex-db-dev@ocioededatalakedbr.dfs.core.windows.net/delta/events/"])
 
-dbutils.widgets.dropdown("output_database", "ocio_dex_dev", ["ocio_dex_dev"])
+####### this can be used if final gold moves to Edav, etc..
+dbutils.widgets.dropdown("gold_output_database", "ocio_dex_dev", ["ocio_dex_dev"])
+dbutils.widgets.dropdown("gold_output_database_checkpoint_prefix", "abfss://ocio-dex-db-dev@ocioededatalakedbr.dfs.core.windows.net/delta/events/", ["abfss://ocio-dex-db-dev@ocioededatalakedbr.dfs.core.windows.net/delta/events/"])
 
-dbutils.widgets.dropdown("output_checkpoint_prefix", "abfss://ocio-dex-db-dev@ocioededatalakedbr.dfs.core.windows.net/delta/events/", ["abfss://ocio-dex-db-dev@ocioededatalakedbr.dfs.core.windows.net/delta/events/"])
+# COMMAND ----------
+
+database =  dbutils.widgets.get("database")
+database_checkpoint_prefix = dbutils.widgets.get("database_checkpoint_prefix")
+
+gold_output_database =  dbutils.widgets.get("gold_output_database")
+gold_output_database_checkpoint_prefix = dbutils.widgets.get("gold_output_database_checkpoint_prefix")
 
 # COMMAND ----------
 
-input_database =  dbutils.widgets.get("input_database")
+# MAGIC %md
+# MAGIC ### Common Functions
 
-output_database =  dbutils.widgets.get("output_database")
+# COMMAND ----------
 
-output_checkpoint_prefix = dbutils.widgets.get("output_checkpoint_prefix")
+def normalizeString(str):
+    return str.replace("-", "_").lower()
+
+def normalize(name):
```

Comment:
If you return str(name) when name is None, then you will return "None". Is that what you intended?